app/src/main/res/values/styles.xml

```diff
@@ -47,7 +50,8 @@
     </style>
 
     <style name="TextInputLayoutBase" parent="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox.Dense">
-        <item name="android:titleTextColor" tools:targetApi="m">?android:attr/textColorPrimary</item>
+        <item name="android:titleTextColor" tools:targetApi="m">?android:attr/textColorPrimary
+        </item>
```

Comment:
Accidental newline?

Reply:
It must have been caused by android formatter because it crosses the hard wrap guide